apps/web/tsconfig.server.json

```diff
@@ -7,5 +7,5 @@
     "tsBuildInfoFile": "../../tmp/buildcache/apps/web/server",
     "types": ["node"]
   },
-  "include": ["server.ts"] //"include": ["server.ts", "**/*.js", "**/*.ts", "**/*.tsx", "next-env.d.ts"]
+  "include": ["server.ts", "**/*.js", "**/*.ts", "**/*.tsx", "next-env.d.ts"]
```

Comment:
If we change this then the build fails on the jest tests. Anyone know why? @kksteini @kksteini

Reply:
I think we should revert these changes in particular. The bad thing about reverting them is that in development, the strict failures will be silently ignored.

---

But to answer that question as best I can:
Do you mean, if we extend the include block?
I haven't looked too much into this but as I understand it, the recent upgrade made these changes because a server is being bootstrapped (or something please no pedantry :sweat_smile: ) for these apps.
This tsconfig.server.json is an extension on the base tsconfig for the app.

This include block in this extension overrides the previous block for the build.
This means that the tsconfig, when extended, stops looking at "**/*.js, "**/*.ts", and tsx and so on.

So I imagine that the tests had strict failures in them that were being ignored by the include block override.

Or am I overthinking this? :laughing: